Signal/src/Loki/View Controllers/NewClosedGroupVC.swift

```diff
@@ -169,7 +169,10 @@ final class NewClosedGroupVC : UIViewController, UITableViewDataSource, UITableV
         }
         let userHexEncodedPublicKey = getUserHexEncodedPublicKey()
         let members = [String](selectedContacts) + [ userHexEncodedPublicKey ]
```

Comment:
`userHexEncodedPublicKey` is wrong here, we should be adding the master device to the members.

Basically **ALL** closed group logic is centered around the master device, a slave/secondary device public key should never appear in members/admin.

What we should do instead is upon receiving a group message, we should map the sender to their master device and use that to check if they are in the group / admin.